imports/plugins/core/orders/client/components/orderTable.js

```diff
@@ -371,7 +372,7 @@ class OrderTable extends Component {
           />
         }
         <SortableTable
-          tableClassName={`rui order table -highlight ${this.props.selectedItems.length > 0 ?
+          tableClassName={`rui order table -striped -highlight ${this.props.selectedItems.length > 0 ?
```

Comment:
Whats the purpose of the `-`(dash) in front of the class names for `striped` and `highlight`. Seems unnecessary to me? @kieckhafer thoughts?

Reply:
That's just how ReactTable does it, it's not on our end.